### DISCUSSION TITLE
[Solved] GLPI agent installation on windows not working

### DISCUSSION THREAD

**Discussion by franckneo** [glpi-project/glpi-agent#761] [Q&A] [answered]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

Hello,
We were using GLPI agent 1.4, saw the 1.10 is available, add a GPO : 
msiexec /i C:\GLPI-Agent-1.10-x64.msi /quiet /log "\path\log.log" SERVER=http://<IP>/glpi/plugins/fusioninventory/

Try to push the upgrade to only 1 UO first, some are working fine and some...not.
In the 127.0.0.1:62354 : no reply
From GLPI server trying to refresh status : no reply.
BUT : the installation said no error : MSI (s) (9C:6C) [13:36:08:497]: Product: GLPI Agent 1.10 -- Installation completed successfully.
MSI (s) (9C:6C) [13:36:08:497]: Windows Installer a installé le produit. Nom du produit : GLPI Agent 1.10. Version du produit : 1.10. Langue du produit : 1033. Fabricant : Teclib'. Réussite de l’installation ou état d’erreur : 0.

The service is running just fine.

Can fail on Windows 11 23H2 or Win 10 22H2.

What kind of log can be relevant for it?
Thank you in advance for any reply.
Already tried to uninstall and reinstall the agent but same issue.

### To reproduce

Upgrade from 1.4 to 1.10?
Not sure, because some are working fine and report with agent 1.10 in the server.

### Expected behavior

I expected the web server to start and client to be "checked"

### Operating system

Windows

### GLPI Agent version

v1.10

### GLPI version

9.5.x (See additional context below)

**Comment by g-bougard** [comment]
Hi @franckneo 

I move your issue to Q&A discussions as you're requesting assistance not reporting a glpi-agent bug.

**Comment by g-bougard** [comment, marked as answer]
You are using a GPO to directly install glpi-agent, but you should definitively use the official vbs script, even in a GPO as it checks msi-server is really available, see https://glpi-agent.readthedocs.io/en/latest/installation/index.html#large-installations

**Reply by franckneo** [reply]
Thank you for this part, GPO deleted and will install the rest with this script.

**Comment by franckneo** [comment]
Je vois que vous êtes de Montpellier...pourquoi s’embêter a parler anglais si on est 2 français...
Le soucis semble résolu en réinstallant via le script VBS. Je ne sais pas pourquoi l'upgrade manuel déconnait, mais soucis résolu.
Encore merci pour le script.

**Reply by g-bougard** [reply]
Hi @franckneo 

we speak english on this project as people requests community support, my answer can also help anybody around the world and of course anyone from our community would be able to answer, not just me.

If you want a personalized french support, [use a GLPI subscription](https://services.glpi-network.com/#offers).

**Reply by franckneo** [reply]
So I will ask in english, with VBS it works perfectly...on my test desk.
Using the company network, I got an error :
line 380, car 4 :
   Set web = Nothing
   Set web = CreateObject("WinHttp.WinHttpRequest.5.1")
   If web Is Nothing Then Set web = CreateObject("WinHttp.WinHttpRequest")
   If web Is Nothing Then Set web = CreateObject("MSXML2.ServerXMLHTTP")
   If web Is Nothing Then Set web = CreateObject("Microsoft.XMLHTTP")
   web.Open "GET", strURL, False
>   web.Send

Timeout
Code 80072EE2
Source WinHttp.WinHttpRequest

Likely our proxy blacklist. I will add in our whitelist : 
https://github.com/glpi-project/
https://nightly.glpi-project.org/
Do you think it will be enough (because I need to wait the night to whitelist update).

Thank you

**Reply by g-bougard** [reply]
Only `https://github.com/glpi-project/` is required, nightly url is only required if you plan to test nightly builds.

And yes, this would be enough.
